### ISSUE TITLE
Adding a non guild member

### ISSUE THREAD

**Issue by tomr578uk** [Roeshambo/MonolithDKP#336]
My guild raids with another guild,and used monolith for the first time between the guild last night. Everybody apart from one added with wither the add raid members,or using the target option.
But one person could not be added in any way at all,he even tried unistalling and re-installing,and even a full pc restart did nothing.
Have used it before with non-guild members and never had this problem.
Any help on this would be appreciated,thanks

**Comment by ghost** [comment]
The only workaround I know at the moment is:
 
1. Player joins the guild
2. You add him to the Database
3. You sync to all officers
4. Player leaves the guild and joins his old guild

Then he will still be in your database, the only problem is that he won't be able to see/access his DKP.

**Comment by tomr578uk** [comment]
Ok thanks. I will speak to him and give this a go



Sent from Samsung Mobile on O2


-------- Original message --------

**Comment by tomr578uk** [comment]
Hi. We tried the workaround that you suggested and no luck. He disabled all his other addons to see if that was what was causing the problem, and still no joy.

Next he switched to another character that is in his guild as well to see if that one would show up, and it worked immediately, as everyone else has done.

From this, we feel it is more a problem on the Blizzard side with that character in particular, so he is going to get them to check if there is any corruption on their side for that.

As soon as we know anything, I will let you know in case someone else has a similar problem, but thanks for your help anyway.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by marknl** [comment]
Sounds like a character name issue? Does the affected player name have any weird characters in it? Anything that's not "a-z"

**Comment by tomr578uk** [comment]
Thanks for replyingNothing in the character name,just normal letters.Before we actually starting raiding together,I went into a dungeon with him,to show him how it worked,added him on no problem.Then we reset everything for a fresh start for the raids,everyone else added,but we couldn't get him on. If he whispers me to see how much dkp he has,it still says he has the amount I gave him for the test.As it stands,it isn't a problem if it is just him,we are keeping a manual record for him. If it started affecting more people it may be an issue,but as of now,no other problemsSent from Samsung Mobile on O2
-------- Original message --------From: Marknl <notifications@github.com> Date: 19/03/2020  10:15  (GMT+00:00) To: Roeshambo/MonolithDKP <MonolithDKP@noreply.github.com> Cc: tomr578uk <tom__rowe@hotmail.co.uk>, Author <author@noreply.github.com> Subject: Re: [Roeshambo/MonolithDKP] Adding a non guild member (#336) 
Sounds like a character name issue? Does the affected player name have any weird characters in it? Anything that's not "a-z"

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Roeshambo/MonolithDKP/issues/336#issuecomment-601097450",
"url": "https://github.com/Roeshambo/MonolithDKP/issues/336#issuecomment-601097450",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by marknl** [comment]
Yeah i think the addon keeps historical data .. so you should've wiped the table, deleted the lua files in saved variables or something to really make sure all old data is gone. Then start from scratch.

Guess that's not an option anymore right now.

**Comment by tomr578uk** [comment]
Unless it is his side that needs all that done. Will try it and see how it goes with that thanksSent from Samsung Mobile on O2
-------- Original message --------From: Marknl <notifications@github.com> Date: 19/03/2020  11:05  (GMT+00:00) To: Roeshambo/MonolithDKP <MonolithDKP@noreply.github.com> Cc: tomr578uk <tom__rowe@hotmail.co.uk>, Author <author@noreply.github.com> Subject: Re: [Roeshambo/MonolithDKP] Adding a non guild member (#336) 
Yeah i think the addon keeps historical data .. so you should've wiped the table, deleted the lua files in saved variables or something to really make sure all old data is gone. Then start from scratch.
Guess that's not an option anymore right now.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Roeshambo/MonolithDKP/issues/336#issuecomment-601119384",
"url": "https://github.com/Roeshambo/MonolithDKP/issues/336#issuecomment-601119384",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
